monitoring/tests/unit/test_query.py

```diff
@@ -127,14 +146,13 @@ def test_constructor_maximal(self):
         self.assertEqual(query._start_time, T0)
         self.assertEqual(query._end_time, T1)
 
-        self.assertIsNone(query._per_series_aligner)
-        self.assertIsNone(query._alignment_period_seconds)
-        self.assertIsNone(query._cross_series_reducer)
+        self.assertEqual(query._per_series_aligner, 0)
+        self.assertEqual(query._alignment_period_seconds, 0)
+        self.assertEqual(query._cross_series_reducer, 0)
         self.assertEqual(query._group_by_fields, ())
 
     def test_constructor_default_end_time(self):
-        import datetime
-        import mock
+        from google.cloud.monitoring_v3 import MetricServiceClient
 
         MINUTES = 5
         NOW, T0, T1 = [
```

Comment:
This appears to be an old line, but it is odd that this test use multiple assignment and not three single assignments.
```
NOW = datetime.datetime(...)
T0 = datetime.datetime(...)
T1 = datetime.datetime(...)
```

Is there any rationale of building the array and extracting to the tuple?

Reply:
Fixed.